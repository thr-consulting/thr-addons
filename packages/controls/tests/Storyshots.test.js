import initStoryshots from '@storybook/addon-storyshots';
import ShallowRenderer from 'react-test-renderer/shallow';

const shallowSnapshot = ({story, context}) => {
	expect(new ShallowRenderer().render(story.render(context))).toMatchSnapshot();
};

initStoryshots({
	test: shallowSnapshot,
});
