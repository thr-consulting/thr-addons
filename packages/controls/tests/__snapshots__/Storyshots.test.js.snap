// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots MaskedInput default 1`] = `
<Grid>
  <GridRow>
    <GridColumn
      width={4}
    >
      <Form
        as="form"
      >
        <FormField>
          <label>
            Masked Input
          </label>
          <MaskedInput
            mask={
              Object {
                "mask": "A9A-9A9",
              }
            }
            onBlur={[Function]}
            onChange={[Function]}
            value={null}
          />
        </FormField>
      </Form>
    </GridColumn>
  </GridRow>
</Grid>
`;

exports[`Storyshots TForm default 1`] = `
<div
  className="ui grid"
>
  <GridRow>
    <GridColumn
      width={4}
    >
      <TForm
        initialValues={
          Object {
            "text": "fdafa",
          }
        }
        onSubmit={[Function]}
        render={[Function]}
      />
    </GridColumn>
  </GridRow>
</div>
`;
